geometry_msgs/Pose object_position
int64 place_point
---
bool is_successful